test/JDBC/upgrade/latest/schedule

```diff
@@ -529,4 +529,7 @@ babel_test_int8_numeric_oper
 babel_test_int2_numeric_oper
 login_token-dep
 test_like_for_AI
-BABEL-4869
\ No newline at end of file
+BABEL-4869
+babel_4328_datetime
+babel_4328_datetime2
+babel_4328_datetimeoffset
```

Comment:
We also need to add these tests to `15_7/schedule` and `14_12/schedule` files when we cherry-pick this to previous versions. So, let's raise the PRs for 3.x and 2.x and then we will merge them from 2.x to 4.x in that order.